From 95fde3fbb7705221e4d799c5ebeb59d7d0f2f8e9 Mon Sep 17 00:00:00 2001
From: David Garske <david@wolfssl.com>
Date: Thu, 22 Dec 2022 17:01:19 -0800
Subject: [PATCH] Merge pull request #5911 from
 julek-wolfssl/DtlsMsgPoolSend-sendSz

DtlsMsgPoolSend: Use correct sendSz
(cherry picked from commit 62e3835b0e278d245ae7fb1805f4f2b31e6dca04)
---
 src/internal.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/internal.c b/src/internal.c
index c58ae3a1e..8d6fd0909 100644
--- a/src/internal.c
+++ b/src/internal.c
@@ -8857,9 +8857,8 @@ int DtlsMsgPoolSend(WOLFSSL* ssl, int sendOnlyFirstPacket)
 #endif
 
 
-                /* add back in header space from saved pool size */
-                sendSz += DTLS_HANDSHAKE_EXTRA;
-                sendSz += DTLS_RECORD_EXTRA;
+                /* add back in record header space from saved pool size */
+                sendSz += DTLS_RECORD_HEADER_SZ;
 
                 if ((ret = CheckAvailableSize(ssl, sendSz)) != 0) {
                     WOLFSSL_ERROR(ret);
-- 
2.39.1

